### PR TITLE
docs: add first release checklist

### PR DESCRIPTION
**Pull request by diasdavid**
@dignifiedquire @VictorBjelkholm @haadcode @RichardLitt what else should we include in the release check list?

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+> List of actions to perform before issuing a new js-ipfs release
+
+- [ ] Ensure that all tests are passing, this includes:
+  - [ ] unit
+  - [ ] interop
+  - [ ] sharness
+- [ ] Run tests of the following projects with the new release:
+  - [ ] orbitdb
+  - [ ] ipfs-log
+- [ ] Announcements (both pre-release and post-release)
```
Comment (dignifiedquire) [inline RELEASE.md]: This should include a standard message + account we are using.

**Comment by RichardLitt** [comment]
Update README?

**Comment by dignifiedquire** [comment]
> Update README?

with what changes?

**Comment by RichardLitt** [comment]
Thought there was a version hardcoded in it. Doesn't look like it. Yeah, I can't think of anything to add.

**Comment by VictorBjelkholm** [comment]
"Make sure changelog is updated" feels like something that is missing.

We should clearly define that "issuing a new js-ipfs release" means publishing to npm.

Once we have cross-platform workers in jenkins, we should also make sure that the tests passes on all the supported platforms and nodejs versions.

Maybe making sure the README.md is always up-to-date makes sense to put in the release checklist as well.

File: RELEASE.md (hunk 1 above)
Comment (diasdavid) [inline RELEASE.md]: 👍

**Comment by diasdavid** [comment]
- [ ] add orbit-core to the list

**Comment by diasdavid** [comment]
There we go :)

**Pull request merged**